pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -965,4 +979,60 @@ public String getLoadManagerClassName() {
     public void setLoadManagerClassName(String loadManagerClassName) {
         this.loadManagerClassName = loadManagerClassName;
     }
+
+    public String getActiveVersion() {
+        return activeVersion;
+    }
+
+    public void setActiveVersion(String activeVersion) {
+        this.activeVersion = activeVersion;
+    }
+
+    private String getStringFromResource(String resource, String attribute, String defaultValue) {
+        try {
+            InputStream inputStream = getClass().getClassLoader().getResourceAsStream(resource);
```

Comment:
Using a property file, we don't need to repeat the parsing logic. Take a look at `HttpClient.getPulsarClientVersion()` where the same thing is being done to report the client version.